src/state/actions/dispatchArNSUpdate.ts

```diff
@@ -51,5 +86,9 @@ export function dispatchArNSUpdate({
     });
   });
 
-  emitter.fetchProcessesOwnedByWallet({ address: walletAddress.toString() });
+  emitter
+    .fetchProcessesOwnedByWallet({ address: walletAddress.toString() })
+    .catch((e) =>
+      errorHandler('Error getting processes owned by wallet: ' + e.message),
```

Comment:
Would it be clearer to the user to mention "ANT processes"?

Reply:
Specified assets instead sinces both domains and ants are affected